import_export/fields.py

```diff
@@ -116,7 +121,11 @@ def save(self, obj, data, is_m2m=False, **kwargs):
                 if not is_m2m:
                     setattr(obj, attrs[-1], cleaned)
                 else:
-                    getattr(obj, attrs[-1]).set(cleaned)
+                    if self.m2m_add:
+                        new_values = [val for val in cleaned if val not in getattr(obj, attrs[-1]).all()]
```

Comment:
The [docs](https://docs.djangoproject.com/en/dev/ref/models/relations/#django.db.models.fields.related.RelatedManager.add) imply this call is not needed (?)

> Using add() on a relation that already exists won’t duplicate the relation, but it will still trigger signals.

Reply:
Good catch.  I missed that in the docs.  I've removed this and I do test for it so it is unnecessary.